### PR TITLE
docs: add information where web server is listening

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

File: docs/development/getting-started.rst [hunk 1]
```diff
@@ -59,6 +59,9 @@ The example data located in ``dev/example.sql.xz`` is taken from
 anything private. The password for every account has been set to the string
 ``password``.
 
+Type `docker ps` to see ip and port for web container to access warehouse webserver.
```
Comment (dstufft) [inline docs/development/getting-started.rst]: I don't think this actually works if you're using boot2docker from an OSX machine, this is what I get:

```console
$ docker ps
CONTAINER ID        IMAGE                   COMMAND                CREATED             STATUS              PORTS                    NAMES
72b922351160        warehouse_web:latest    "python -m warehouse   3 seconds ago       Up 1 seconds        0.0.0.0:80->8000/tcp     warehouse_web_1
310cfc930f06        warehouse_camo:latest   "node server.js"       4 seconds ago       Up 3 seconds        0.0.0.0:9000->9000/tcp   warehouse_camo_1
534fe513b9db        redis:latest            "/entrypoint.sh redi   6 seconds ago       Up 5 seconds        6379/tcp                 warehouse_redis_1
5da892994d38        postgres:9.4            "/docker-entrypoint.   7 seconds ago       Up 6 seconds        5432/tcp                 warehouse_db_1
```

In order to find the IP address I need to connect to, I have to run ``boot2docker ip``.

**Comment by domenkozar** [comment]
@dstufft hopefully fixed, I also added a section about db migrations.

File: warehouse/legacy/api/index.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# limitations under the License.
+
+from pyramid.view import view_config
+
+
+@view_config(
+    route_name="legacy.api.index",
+    renderer="legacy/api/index.html",
+)
+def index(request):
```
Comment (dstufft) [inline warehouse/legacy/api/index.py]: Did you mean to commit this file?

**Comment by domenkozar** [comment]
@dstufft nope, fixed.

**Comment by dstufft** [comment]
LGTM, will merge once tests pass.

**Comment by domenkozar** [comment]
@dstufft passes now

**Pull request merged**